test/features/support/hooks.js

```diff
@@ -0,0 +1,38 @@
+/* jshint node: true */
+
+'use strict';
+
+var myHooks = function () {
+
+  /**
+   * BeforeFeatures handler
+   *
+   * Provides access to the pageObjectsMap
+   * on browser
+   *
+   * @params {event} event The event being attached to
+   * @params {function} callback The callback function of the hook
+   */
+  this.registerHandler('BeforeFeatures', function (event, callback) {
+    browser.pageObjects = require('./pageObjectMap.js');
```

Comment:
If you look at the code in that page, though, it doesn't refer to “page objects”, it just calls them `something_page` or `SomethingPage`. You wouldn't call an object that follows the [singleton pattern](https://en.wikipedia.org/wiki/Singleton_pattern) `SomethingSingleton` (unless you were writing Java, I guess), so I think `pageObject` is wrong: you're leaking the name of the pattern into your code.